src/test/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookTriggerImplTest.java

```diff
@@ -115,12 +115,12 @@ public void testReregisterWebhook() {
                 .register(
                         argThat(args -> args.equals(BITBUCKET_BASE_URL)),
                         any(GlobalCredentialsProvider.class),
-                        argThat(arg -> arg.equals(repo)), eq(true), eq(false));
+                        argThat(arg -> arg.equals(repo)), eq(false), eq(true));
```

Comment:
because I swapped the order to be alphabetical there are a few of these, I figured it was better to keep consistent and alphabetical even if it makes this PR a smidgen more annoying to review. Sorry

Reply:
Looks like it was a lot more annoying to write than it is to review!